documentation/antora.yml

```diff
@@ -10,6 +10,7 @@ asciidoc:
     debezium-dev-version: '1.3'
     debezium-kafka-version: '2.6.0'
     debezium-docker-label: '1.2'
+    DockerKafkaConnect: registry.redhat.io/amq7/amq-streams-kafka-25:1.5.0
```

Comment:
Isn't this variable only used in ifdef blocks related to downstream?

Reply:
Yes, I think that is correct. 
Is there a problem with having it here?
If yes, we don't need it in the antora file.

Reply:
There isn't a problem having it other than the _maintenance_ aspect.

Reply:
Please do whatever you think best.
If you want me to do something, please let me know. 
As soon as this PR is merged and backported to 1.2, I will fetch the updates and rebuild the downstream UG.